docs/user-guide/compute-resources.md

```diff
@@ -203,6 +203,10 @@ Containers:
       memory:       50Mi
     State:      Running
       Started:      Tue, 07 Jul 2015 12:54:41 -0700
+		Last Termination State:     Terminated
+			Exit Code:                137
```

Comment:
> State and Last Terminations State are aligned for me

Looks like the existing example uses spaces and your addition uses tabs, which is why it looks aligned to you but not for others. Can you change your addition to uses spaces?